pkg/domain/entities/secrets.go

```diff
@@ -12,6 +12,7 @@ type SecretCreateReport struct {
 
 type SecretCreateOptions struct {
 	Driver string
+	Opts   map[string]string
```

Comment:
I think this should be consistent throughout
```suggestion
	DriverOpts   map[string]string
```

Reply:
changed.